content/doc/book/system-administration/monitoring.adoc

```diff
@@ -69,16 +69,9 @@ https://issues.jenkins.io/browse/JENKINS-25695[JENKINS-25695].
 Sometimes, for example https://wiki.jenkins.io/display/JENKINS/Remoting+issue[to diagnose the agent connection loss
 problem], you may want to disable the ping thread.  This needs to be done in two places.
 
-To disable the controller from pinging agents, the system property `+hudson.slaves.ChannelPinger.pingIntervalSeconds+` should
-be set on the controller JVM to -1. You can also change the value in memory for a running Jenkins, if you don't want to
-restart Jenkins *(Deprecated since 2.37)*:
+Disable the controller from pinging agents by `+hudson.slaves.ChannelPinger.pingIntervalSeconds+` on the controller JVM to -1.
 
-[source,groovy]
-----
-Jenkins.instance.injector.getInstance(hudson.slaves.ChannelPinger.class).@pingInterval = -1
-----
-
-Since 2.37, set pingIntervalSeconds and pingTimeoutSeconds on the controller JVM to -1:
+You can also change the value in memory for a running Jenkins, if you don't want to restart Jenkins. Set pingIntervalSeconds and pingTimeoutSeconds on the controller JVM to -1:
```

Comment:
```suggestion
You can also change the value of memory for a running Jenkins, if you don't want to restart Jenkins.
Set `pingIntervalSeconds` and `pingTimeoutSeconds on the controller` JVM to -1:
```
small suggestion to change in to of (disregard if this changes the messaging)
suggestion to format values/strings